addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiSensorHtHandler.java

```diff
@@ -23,6 +25,8 @@
  */
 public class XiaomiSensorHtHandler extends XiaomiSensorBaseHandler {
 
+    private final Logger logger = LoggerFactory.getLogger(XiaomiSensorHtHandler.class);
```

Comment:
Do we need the `logger` in this class? Or why did you add it?

Reply:
This was added by mistake. Removed it now